ghost/core/core/server/services/milestone-emails/index.js

```diff
@@ -0,0 +1,71 @@
+let hasScheduled = false;
+
+/**
+ * @param {'arr'|'members'} milestoneType
+ *
+ * @returns {Promise<any>}
+ */
+module.exports = async (milestoneType) => {
+    // const labs = require('../../../shared/labs');
+
+    // if (labs.isSet('milestoneEmails')) {
+    const db = require('../../data/db');
+    const MilestoneQueries = require('./MilestoneQueries');
+
+    const {
+        MilestonesEmailService,
+        InMemoryMilestoneRepository
+    } = require('@tryghost/milestone-emails');
+    const config = require('../../../shared/config');
+    const milestonesConfig = config.get('milestones');
+    const {GhostMailer} = require('../mail');
+
+    const mailer = new GhostMailer();
+    const repository = new InMemoryMilestoneRepository();
+    const queries = new MilestoneQueries({db, milestonesConfig});
+
+    const milestonesEmailService = new MilestonesEmailService({
+        mailer,
+        repository,
+        milestonesConfig, // avoid using getters and pass as JSON
+        queries
+        // TODO: do we need to check if Stripe is live enabled?
+    });
+
+    return await milestonesEmailService.checkMilestones(milestoneType);
```

Comment:
Not sure if this is the right way to go, because when using it as in-memory storage it gets instantiated every time again, which doesn't make sense I think....